huxley/www/js/components/AdvisorProfileView.js

```diff
@@ -12,6 +12,7 @@ var InnerView = require('components/InnerView');
 var LogoutButton = require('components/LogoutButton');
 var ConferenceContext = require('components/ConferenceContext');
 var CurrentUserActions = require('actions/CurrentUserActions');
+var MarkdownTemplate = require('components/MarkdownTemplate');
```

Comment:
nit: Maybe `TextTemplate`, because the component doesn't actually know or care that the text is in markdown format; it only cares that it has the template variables in there.